benchmarks/vector_db_compare/minima_qdrant_runner.py

```
@@ -0,0 +1,1735 @@
+        assert self.client is not None
+        accumulator, offset = StateAccumulator(), None
+        mismatches, maximum_delta = 0, 0.0
+        while True:
+            rows, offset = self.client.scroll(collection_name=self.collection, limit=self.config["batch_size"], offset=offset,
+                with_payload=True, with_vectors=[self.config["vector_field"]], timeout=self.operation_timeout)
+            for row in rows:
+                payload, vectors = getattr(row, "payload", None) or {}, getattr(row, "vector", None)
+                vector = vectors.get(self.config["vector_field"]) if isinstance(vectors, dict) else None
+                document = {**payload, "vector": vector}
+                if set(document) != set(DOCUMENT_KEYS) or not isinstance(vector, list) or len(vector) != self.config["dimension"]:
+                    raise RuntimeError("Qdrant scroll row does not match the Minima document schema")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)